src/Analyzers/Core/CodeFixes/GenerateConstructor/GenerateConstructorHelpers.cs

```diff
@@ -345,25 +346,28 @@ private static bool IsSymbolAccessible(ISymbol? symbol, SemanticDocument documen
     private static bool IsViableFieldOrProperty(
```

Comment:
note: we havea  few copies of this floating around, all slightly different.  end goal in followup is to unify all of them.